modules/ROOT/pages/scheduler-concept.adoc

```diff
@@ -3,18 +3,18 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-The Scheduler is a flow event source that is configured to trigger at certain times to execute the Scheduler's flow. A Scheduler is either configured with a fixed repeating triggering *frequency* (such as every second), or with a more general cron *expression*. A cron expression can specify one or more specific dates and times, but more often specifies recurring dates, times, or intervals (for example the cron expression `* * * ? * *` means to trigger every second). 
-
-The flow can be configured to only allow the Scheduler event source to execute one flow instance at a time, or can allow concurrent flow instances to execute. 
+The Scheduler component is an event source that initiates execution of a flow at configured times. You can configure a fixed and repeating frequency (such as every second) or a cron expression. A cron expression can specify one or more specific dates and times, but more commonly, the expression specifies recurring dates, times, or intervals. For example, the cron expression `* * * ? * *` configures a Scheduler to start its flow every second.
 
 Consider the following when you implement a Scheduler in your Mule application:
 
 * Schedulers use the same timezone as the machine on which Mule is running. However, if an application is running in CloudHub, the Scheduler conforms to the UTC timezone, regardless of the geographic region in which the application is running.
-// TODO: MULE-14438
-* If xref:execution-engine.adoc#backpressure[back-pressure] is triggered because no resources are available at the time of the trigger, that trigger is skipped. In this case a *MULE:FLOW_BACK_PRESSURE* error is thrown indicating the flow cannot accept new events at this time. In Anypoint Studio, this error is only logged once the first time it occurs. You can capture log messages each time the Scheduler is triggered, including events that do not execute the flow, by enabling debug level logging in the Mule application.  
+* If xref:execution-engine.adoc#backpressure[back pressure] occurs because Mule lacks sufficient resources to process an event in the flow, the application skips the scheduled trigger for the event and throws a MULE:FLOW_BACK_PRESSURE error. This error indicates that the flow cannot accept new events at that time. Anypoint Studio logs this error only once, the first time it occurs. The error is informational and does not prevent the subsequent executions of the flow.
```

Comment:
should somebody from studio team verify this final sentence as well?

Reply:
removed sentence